### DISCUSSION TITLE
api error api-logo.svg

### DISCUSSION THREAD

**Discussion by chuckkay** [vladmandic/sdnext#686] [General]
anyone know why im getting this error in startup... everything seems to work fine but dont understand why:

API error: GET: http://localhost:7860/static/img/api-logo.svg {'error': 'HTTPException', 'detail': 'Forbidden', 'body': '', 'errors': '403'}

**Comment by vladmandic** [comment]
it comes and goes away sometimes, magic of new gradio. if their api docs js module loads before main page is ready, it will report that.
and file itself is used in the new api docs page (which is much worse than old one).
used to be <http://127.0.0.1/docs>, now its <https://127.0.0.1:7860/?__theme=dark&view=api>

**Reply by chuckkay** [reply]
thanks, good to know it's not me

**Reply by brianjz** [reply]
Wow, you're not kidding. That new page is hot garbage.
